<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constructive Solid Geometry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>10</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="abdiffRadio">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Difference (A-B)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="badiffRadio">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Difference (B-A)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="intersectRadio">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>105</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Intersect</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="unionRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Union</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="GLWidget" name="glpanel" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>561</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>70</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QWidget" name="scalePanel" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="scaleButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>7</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sx">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>9</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sy">
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>8</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sz">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>8</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>12</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>11</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>z :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="translatePanel" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="translateButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>7</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Translate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tx">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>9</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ty">
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>8</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tz">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>8</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>12</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>11</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>z :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <addaction name="action3D_Line"/>
     <addaction name="action3D_Triangle"/>
     <addaction name="actionCuboid"/>
     <addaction name="actionCylinder"/>
     <addaction name="actionCone"/>
     <addaction name="actionSphere"/>
    </widget>
    <addaction name="actionOpen_File"/>
    <addaction name="menuAdd"/>
    <addaction name="action_Save_Result"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCurrent_Object"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="action_Save_Result">
   <property name="text">
    <string>&amp;Save Result</string>
   </property>
  </action>
  <action name="action3D_Line">
   <property name="text">
    <string>3D Line</string>
   </property>
  </action>
  <action name="action3D_Triangle">
   <property name="text">
    <string>3D Triangle</string>
   </property>
  </action>
  <action name="actionCuboid">
   <property name="text">
    <string>Cuboid</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionCurrent_Object">
   <property name="text">
    <string>Current Object</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
